Discovery v2 - hightlight port change (#2091)
neo4j/docs-operations: modules/ROOT/pages/clustering/setup/discovery.adoc

Possible values are `V1_ONLY`, `V2_ONLY`, `V1_OVER_V2`, and `V2_OVER_V1`.
* xref:configuration/configuration-settings.adoc#config_dbms.kubernetes.discovery.v2.service_port_name[`dbms.kubernetes.discovery.v2.service_port_name`] to specify the name of the service port name used in the Kubernetes API when using version v2.

> xref:configuration/ports.adoc[`Ports`] have also changed in discovery v2.

Reviewer: ```suggestion
xref:configuration/ports.adoc[`Ports`] have also changed in discovery service v2.
```

> xref:configuration/ports.adoc[`Ports`] have also changed in discovery service v2.

Reviewer: ```suggestion
xref:configuration/ports.adoc#_cluster[`Ports`] have also changed in discovery service v2.
```

> Cluster discovery v1 port is no longer in used (port 5000).

Reviewer: ```suggestion
Port `5000`, the discovery v1 port, is no longer used.
```

> Instead discovery v2 now uses the Cluster internal traffic port (port 6000).

Reviewer: ```suggestion
Instead, discovery service v2 uses port `6000` - the cluster internal traffic port.
```
